File: Linked_Pila_Cola/Nodo.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Nodo.cpp
 * Author: Juan Pablo Martinez
 * 
 * Created on 25 de septiembre de 2018, 19:27
 */

#include "Nodo.h"
#include <cstdlib>

Nodo::Nodo(int dato) {
    this->dato = dato;
    this->next = NULL;
    this->prev = NULL;
}

int Nodo::getDato() {
    return dato;
}

void Nodo::setDato(int dato) {
    this->dato=dato;
}

Nodo* Nodo::getNext() {
    return next;
}

void Nodo::setNext(Nodo* next) {
    this->next=next;
}

Nodo* Nodo::getPrev() {
    return prev;
}

void Nodo::setPrev(Nodo *prev) {
    this->prev=prev;
}

Nodo::~Nodo() {
}

File: ejempleNodo/src/ejemplenodo/Nodo.java

package ejemplenodo;


public class Nodo {
    private int dato;
    private Nodo next;
    private Nodo prev;
    
    public Nodo(int dato) {
        this.dato = dato;
        next = null;
        prev = null;
        
    }

    public int getDato() {
        return dato;
    }

    public void setDato(int dato) {
        this.dato = dato;
    }

    public Nodo getNext() {
        return next;
    }

    public void setNext(Nodo next) {
        this.next = next;
    }

    public Nodo getPrev() {
        return prev;
    }

    public void setPrev(Nodo prev) {
        this.prev = prev;
    }
    
}

